examples/Advanced_Sampling_Introduction.md

```diff
@@ -0,0 +1,882 @@
+```
+
+```bash id="J7OY5K9VoBBh"
+
+mkdir -p $PYSAGES_ENV .
+unzip -qquo pysages-env.zip -d $PYSAGES_ENV
+```
+
+```bash colab={"base_uri": "https://localhost:8080/"} id="LlVSU_-FoD4w" outputId="85971918-c9b9-44b6-8898-3557ae165cc9"
+apt-cache policy libcudnn8 &> /dev/null
+apt install --allow-change-held-packages libcudnn8=8.4.1.50-1+cuda11.6 &> /dev/null
+update-alternatives --auto libcudnn &> /dev/null
+```
+
+```python id="EMAWp8VloIk4"
+import os
+import sys
+
+ver = sys.version_info
+sys.path.append(os.environ["PYSAGES_ENV"] + "/lib/python" + str(ver.major) + "." + str(ver.minor) + "/site-packages/")
+
+os.environ["XLA_FLAGS"] = "--xla_gpu_strict_conv_algorithm_picker=false"
+os.environ["LD_LIBRARY_PATH"] = "/usr/lib/x86_64-linux-gnu:" + os.environ["LD_LIBRARY_PATH"]
+```
+
+<!-- #region id="we_mTkFioS6R" -->
+## PySAGES
+
+The next step is to install PySAGES.
+First, we install the jaxlib version that matches the CUDA installation of this Colab setup. See the JAX documentation [here](https://github.com/google/jax) for more details.
+<!-- #endregion -->
+
+```bash id="vK0RZtbroQWe"
+
+pip install -q --upgrade pip
+# Installs the wheel compatible with CUDA 11 and cuDNN 8.2 or newer.
+pip install -q --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html &> /dev/null
+```
+
+<!-- #region id="wAtjM-IroYX8" -->
+Now we can finally install PySAGES. We clone the newest version from [here](https://github.com/SSAGESLabs/PySAGES) and build the remaining pure python dependencies and PySAGES itself.
+<!-- #endregion -->
+
+```bash id="B-HB9CzioV5j"
+
+rm -rf PySAGES
+git clone https://github.com/SSAGESLabs/PySAGES.git &> /dev/null
+cd PySAGES
+pip install -q . &> /dev/null
+```
+
+<!-- #region id="KBFVcG1FoeMq" -->
+# Advanced Sampling Methods
+<!-- #endregion -->
+
+```bash id="ppTzMmyyobHB"
+
+mkdir /content/advanced_sampling
+cd /content/advanced_sampling
+```
+
+<!-- #region id="jnMtDUgyXEsr" -->
+Many systems have a separated free-energy landscape, where different basins are divided by energy barriers.
+
+### Examples:
+#### Collective molecular transitions
+* Protein folding
+* Protein ligand binding
+* Liqid membrane fusion
+<!-- #endregion -->
+
+<!-- #region id="mHOcKW-fXoK5" -->
+<img src=https://pubs.acs.org/cms/10.1021/ja910050x/asset/images/large/ja-2009-10050x_0008.jpeg width=500>
+
+<img src=https://pubs.acs.org/cms/10.1021/ja910050x/asset/images/large/ja-2009-10050x_0011.jpeg width=500>
+
+Smirnova, Yuliya G., et al. "Solvent-exposed tails as prestalk transition states for membrane fusion at low hydration." Journal of the American Chemical Society 132.19 (2010): 6710-6718
+<!-- #endregion -->
+
+<!-- #region id="hz7pwhbZXzZp" -->
+#### Phase transisions
+* Liquid-vapor transisition
+* Crystalization
+* Shape transitions
+* Magnetization (Ising model)
+
+![picture](https://upload.wikimedia.org/wikipedia/commons/e/e6/Ising_quench_b10.gif)
+Wikipedia: https://en.wikipedia.org/wiki/Ising_model
```

Comment:
```suggestion
![picture](https://upload.wikimedia.org/wikipedia/commons/e/e6/Ising_quench_b10.gif)

Wikipedia: https://en.wikipedia.org/wiki/Ising_model
```
I think this would be better displayed this way